src/node/performance/worker_service_impl.js

```diff
@@ -36,6 +36,11 @@
 var BenchmarkClient = require('./benchmark_client');
 var BenchmarkServer = require('./benchmark_server');
 
+exports.quitWorker = function quitWorker(call, callback) {
+  callback(null, {});
+  process.exit(0);
```

Comment:
The Node server doesn't have anything like that, but I think it's actually OK anyway. Ideally, though, the driver would only call this after other calls to the same worker have completed.